GOVERNANCE.md

```diff
@@ -0,0 +1,124 @@
+
+For the first formation of the committee, our maintainers nominated a group of people who had great impact or influence on the project, including some members from PingCAP who founded TiDB. New PMC members are selected based on the rules described above.
+
+For a list of the current PMC members, see the [PMC Members](./pmc-list.md).
+
+<!--- need the link to a Guiding Principals (Missions, Values; to be added separately) page as the principals for PMC -->
+
+### Organizer Committee
+
+The Organizer Committee consists of organizers in charge of community event or activity operations. They ensure the execution of PMC’s strategies and decisions. Specifically, the members are the leaders of user groups or community event organizers across regions.
+
+### Developer Group
+
+As the cornerstone of community development, the TiDB Developer Group consists of these roles: Maintainer, Committer, Reviewer, Active Contributor, and Contributor. Each role takes corresponding responsibilities in the community. Collectively they play an important role in the robust development of TiDB. For more details, refer to [Community Membership](TBD). 
+
+The Developer Group operates TiDB projects in two function forms: Special Interest Group (SIG) and Working Group (WG). The diagram below illustrates the definition of the SIG and WG, the internal roles of the group, and the role promotion path.
+
+### Maintainers
+
+While the PMC is the core management group that oversees the project, Maintainers are the technical authority in the Developer Group who function as planners and designers of each project. Maintainers set technical direction and priorities for the sub-project owned by the specific SIG, and ensure its continued health and development. They make or approve design decisions for the project - either directly or by delegating these responsibilities. Maintainers are appointed by the PMC with supermajority votes.
+
+If a Maintainer is no longer interested or cannot perform the Maintainer duties listed above, they should volunteer to be moved to emeritus status.
+
+See [TiDB Maintainers](./maintainer-list.md) for a list of TiDB Maintainers.
+
+#### SIGs
+
+The TiDB project is organized primarily into Special Interest Groups, or SIGs. Each SIG is comprised of members from multiple companies or organizations, with a common purpose of advancing the TiDB project with respect to a specific topic,such as Coprocessor or Documentation. The goal of an SIG is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. Every identifiable subpart of the project (e.g., github org, repository, subdirectory, API, test, issue, PR) is intended to be owned by the corresponding SIG.
```

Comment:
we don't have coprocessor sig in TiDB for now, maybe expression is more correct?